.gitignore

```diff
@@ -2,3 +2,4 @@ build
 composer.lock
 docs
 vendor
+.history
```

Comment:
Remove this line (and add it to your global gitignore https://murze.be/2014/12/create-a-global-gitignore/)